src/web/extractors.rs

```diff
@@ -224,6 +224,26 @@ impl FromRequest for BsoBodies {
                 ));
             }
         };
+
+        // ### debug_client
+        if let Some(uids) = &state.limits.debug_clients {
```

Comment:
This is duplicated below and should be moved to a function, assuming the code will exist for more than a trivial amount of time.

Reply:
Yep. Fortunately, this hack is just to test to see if production mobile clients behave correctly when an OverQuota response is returned, and will be dropped shortly afterward.

It's the reason that the patch also has a really crappy "--debug_client" "--debug_clients" pair of options. That's horribly confusing and should never be allowed (except if it's part of some crappy hack to test clients that can't hit stage)